tests/sentry/monitors/endpoints/test_monitor_ingest_checkin_details.py

```diff
@@ -178,7 +178,14 @@ def test_latest_returns_last_unfinished(self):
             )
 
             path = path_func(monitor.guid, self.latest.guid)
-            resp = self.client.put(path, data={"status": "ok"}, **self.token_auth_headers)
+            resp = self.client.put(
+                path,
+                data={
+                    "status": "ok",
+                    "monitor_config": {"schedule_type": "crontab", "schedule": "* * * * *"},
+                },
+                **self.token_auth_headers,
+            )
```

Comment:
is this the right test case for this update? Should it be it's own test for "checkin update with upsert"?

Reply:
we just want to make sure that sending the `monitor_config` doesn't break anything. it wouldn't be its own test because we are throwing away the data

Reply:
this is the only `latest` test that returns a `200`

Reply:
though I guess this probably affects non `latest` endpoints as well. will put the payload in another passing `200` test

Reply:
Let's call that out in a comment in that case, otherwise it's a bit confusing why we have `monitor_config` in here